src/templates/InstantSearch.js widget/README.md

```diff
@@ -151,3 +158,47 @@ search.addWidgets([
   }),
 ]);
 ```
+
+## Test
```

Comment:
How about we put all the following sections in a "Development" section?

Reply:
agreed, maybe even better in contributing.md?